esx_service/vmdk_ops.py

```diff
@@ -159,17 +159,74 @@ def RunCommand(cmd):
 
     return (s, o)
 
+
+# Translates the vmdk_path into a readable path by
+# removing the UUID string and using tenant_name
+def gets_readable_path(vmdk_path):
+    idx = 0
+    t_name = ''
```

Comment:
what is t_name ? Is it tenant_name ? Then it should be named so. The aqme about t_uuid. There is no reason to save 5 characters in the name and make it harder to read